Source/SVWidgetsLib/Widgets/SVPipelineView.cpp

```diff
@@ -419,7 +428,7 @@ void SVPipelineView::executePipeline()
   //  m_PipelineInFlight = getCopyOfFilterPipeline();
   m_PipelineInFlight = getFilterPipeline();
 
-  emit stdOutMessage("<b>Preflight Pipeline.....</b>");
+  formatStdOutputMessage("Preflight Pipeline.....", true);
```

Comment:
The intent seems to be lost here.  `emit stdOutMessage(` tells me what is being done with the string being passed in without tracking down the functionality.  Even knowing otherwise, reading `formatStdOutputMessage(` feels like it's a local variable or UI feature that will stick around for a bit.  It's a bit long, but I would strongly recommend changing the method name to `emitFormattedStdOutputMessage` or something that describes what the method is doing.